extras/signers.json

```diff
@@ -44,6 +44,19 @@
     "nanexcool": "0x823DF0278e4998cD0D06FB857fBD51e85b18A250",
     "davgarai": "0xAc1aA53108712d7f38093A67d380aD54B562a650"
   },
+  "arb_grant_treasury": {
+    "balancer": {
+      "tritium": "0xcf4fF1e03830D692F52EB094c52A5A6A2181Ab3F",
```

Comment:
I confirm I control this address.  It is registered on other balancer multisigs including the maxi multisig and the dao multisig, and regularly signs and executes balancer transactions.